attribution.md

```diff
@@ -5,11 +5,11 @@ We're grateful to be part of the open source community, and to be able to build
 
 ---
 
-# 3rd-Party Software for [git@github.com:transform-data/metricflow-staging.git]()
+# 3rd-Party Software for [git@github.com:transform-data/metricflow.git]()
 
 
 
-The following 3rd-party software packages may be used by or distributed with **git@github.com:transform-data/metricflow-staging.git**.  Any information relevant to third-party vendors listed below are collected using common, reasonable means.
+The following 3rd-party software packages may be used by or distributed with **git@github.com:transform-data/metricflow.git**.  Any information relevant to third-party vendors listed below are collected using common, reasonable means.
```

Comment:
On line 17 there's an invalid commit sha (`ac34affefc1ff90b740f42098731dad1a540f64b`). We should probably replace it with this one:

`dc975d95150a263d23c74a660f326e35925be662`

I ran FOSSA on that and it seemed to be the same (a few dependencies got dropped out), so I'm comfortable with just this for now. We need to find a better process for managing this file, though.